### PR TITLE
Add emulator for libota_jni.so and strings for Poke4

### PR DESCRIPTION
**Pull request by LeadroyaL**
Decrypt successfully.

I add emulator to obtain keys from libota_jni.so. Please review and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,6 +169,63 @@ Other strings can be got in following steps:
...
+    Connect to device via `adb` as above. Then execute command:
+
+    ```
+    adb pull /system/lib/libota_jni.so
+    ```
+
+    or
+
+    ```
+    adb pull /system/lib/libota_jni.so
```
Comment (nate-moo) [inline README.md:191]: This is the same command as the first one, I think it's supposed to be lib or lib64

Comment (LeadroyaL) [reply README.md:191]: It's a typo. Fixed.

**Comment by Hagb** [comment]
Thanks for your strings and method!
Could you please move the method in the line
https://github.com/Hagb/decryptBooxUpdateUpx/blob/8e141cbfc9bd835a628167e7d1671b44874d625a/README.md?plain=1#L165

to method 2?

And I think we can provide a way to determine the method users should use.
For example, if the above command prints something, the user should use method 2 and use the output as the path of `libota_jni.so`
``` shell
adb shell "ls /system/lib*/libota_jni.so"
```

Sorry for my late reply.

**Comment by LeadroyaL** [comment]
@Hagb done

**Comment by Hagb** [comment]
The PR was rebased (commits about libota_jni.so were squashed) and merged. Thanks for your contribution!

**Pull request merged**